api/src/main/java/com/codeforcommunity/dto/team/TeamMember.java

```diff
@@ -1,11 +1,17 @@
 package com.codeforcommunity.dto.team;
 
+import com.codeforcommunity.dto.blockInfo.Individual;
 import com.codeforcommunity.enums.TeamRole;
 
-public class TeamMember {
-  private int id;
-  private String username;
-  private int blocksCompleted;
-  private int blocksReserved;
-  private TeamRole role;
+public class TeamMember extends Individual {
```

Comment:
I hope this doesn't break the Jackson mapping

Reply:
It's not currently being used by anything (the only method that uses it returns `null` for some reason.